#Essa informação é utilizada para
#Representar um caminho de um determinado robô. Sendo definida como uma lista dinâmica de poses, ela possuir um
#tamanho relativamente grande, possibilitando plotar um caminho bem contínuo. Além de que, não há limitação do uso,
#Assim o desenvolvedor por utilizar a seguência de pontos para outras coisas como: Desenhar áreas de atuação,
#Gradientes e etc. 

uint8 id
d_Pose[] poses
